app/templates/reminders/reminder/reminder-editable.hbs

```diff
@@ -0,0 +1,8 @@
+<p> remember to save your changes! </p>
+<h2 class='display-input'>{{input value=model.title}}</h2>
+<p class='display-date'>{{input value=model.date}}</p>
+<p class='display-notes'>{{input value=model.notes}}</p>
+
+{{#link-to 'reminders' model.id}}
```

Comment:
Ah, I thought we needed it. I guess we don't. That actually makes a lot more sense. It made for some confusion during testing yesterday.